SwiftCharts/ChartCoordsSpace.swift

```diff
@@ -118,7 +174,18 @@ public class ChartCoordsSpace {
                 layer.rect.width + chartSettings.spacingBetweenAxesY
             }, generator: generator)
     }
-    
+
+    /**
+     Uses a generator to make axis layers from axis models. This method is used for all axes.
+
+     - parameter axisModels:    The models used to generate the axis layers
+     - parameter offset:        The offset in points for the generated layers
+     - parameter pointsCreator: TODO: not sure exactly what this effectively does
+     - parameter dimIncr:       TODO: not sure exactly what this effectively does
```

Comment:
Function that calculates space for occupied by axis in muti-axis space, which is the width (in y-axis case) or height (in x-axis case) of axis rect (line + labels) + space between axes.